.changeset/smooth-worms-retire.md

```diff
@@ -0,0 +1,24 @@
+---
+"@clerk/clerk-js": minor
+"@clerk/clerk-react": minor
+"@clerk/types": minor
+---
+
+Extended the `UserButton` component to support custom menu items:
+
+- Introduced `UserButton.MenuItems` as a child component to wrap custom menu items
+- Introduced `UserButton.Link` for custom link items
+- Introduced `UserButton.Action` for custom action items
+- Implemented support for reordering existing items like `manageAccount` and `signOut`
+
+New usage example:
+
+```jsx
```

Comment:
You're missing the closing backticks for this code-block here

Reply:
Fixed